README.md

```diff
@@ -31,7 +36,7 @@ On a local machine:
 3. Set the location of the weather database in the Main.R script of the R_program folder (where it says set database location), along with the name of the weather database (where it says Provide the name of the database in quotes).
```

Comment:
It appears that the `Main.R` script needs to be additionally updated to match the weather database in regards to
`climate.conditions`, `simstartyr`, `endyr`, `climate.ambient`, `deltaFutureToSimStart_yr`, `downscaling.method, `YEARS, etc.
Mentioning that here may be helpful to users

Reply:
@dschlaep you are right - the number of climate.conditions in Main.R needs to correspond to the number of scenarios called when using the generate_rSFSTEP2_structure.sh script. This does deserve additional clarification. Particularly, the number of unique scenarios specified in generate_rSFSTEP2_structure.sh script corresponds to the number of GCMs + current. So if you are using 13 GCMs and current conditions, scenarios = 14 or:

./generate_rSFSTEP2_structure.sh script R_program 1 14 # for 1 site, 13 GCMs + current

I am sorry you weren't able to run this!! I wonder if the issue I describe here was your problem.

Reply:
addressed by https://github.com/DrylandEcology/rSFSTEP2/commit/b1324ec6e41fbada4567efb09acd29e6e3387079